Assets/Developers/Lab/Systems/MovementSystem.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Unity.Entities;
+using Unity.Mathematics;
+using Unity.Transforms;
+using UnityEngine;
+
+using Random = Unity.Mathematics.Random;
+
+namespace JanusVagabond
+{
+  public class MovementSystem : ComponentSystem
+  {
+    const int RANDOM_SEED = 99;
+
+    static Random m_RandomPool = new Random(RANDOM_SEED);
+    protected override void OnUpdate()
```

Comment:
It is an `override` func, but you never call base.OnUpdate(), is there anything important that parent OnUpdate calls?

Reply:
The OnUpdate is called by the main system itself (i.e a kind of scheduler). It's similar to the Update or Start of MonoBehavior.